packages/notification/src/features/utils.ts

```diff
@@ -93,7 +93,7 @@ export function getInformantName(
   if (!name) {
     error(record, 'name not found in informant patient resource')
   }
-  return [name.given?.join(' '), name.family.join(' ')].join(' ').trim()
+  return [name.given?.join(' '), name.family?.join(' ')].join(' ').trim()
```

Comment:
TIL, join removes undefineds and nulls from the source array

Reply:
Ah this works because there are only 2 values in the array, otherwise you would have needed to filter it before joining them. In this case though, `trim` is taking care of the extra spaces